### DISCUSSION TITLE
alist可以给E5账号续期么？

### DISCUSSION THREAD

**Discussion by chouqiao** [AlistGo/alist#3068] [Q&A]
是否可以为[Microsoft 365 E5 Renew Plus账号续期？

**Comment by NovaShen555** [comment]
额？您举个栗子？怎么个实现的

**Reply by chouqiao** [reply]
> 额？您举个栗子？怎么个实现的

我也是小白，因为看到过用OneIndex 给e5续期的教程，原理就是保持api调用吧，  alist同为列表程序，所以我觉得是不是也有这作用。

**Reply by NovaShen555** [reply]
在资源过期时间那里设置一个数字应该可以达到同样的效果吧，这样也可以定期调用api

**Reply by chouqiao** [reply]
> 在资源过期时间那里设置一个数字应该可以达到同样的效果吧，这样也可以定期调用api
不太清楚啊，所以来提问了
